todo_app/templates/index.html

```diff
@@ -1,18 +1,42 @@
 {% extends "layout.html" %}
 {% block title %}To-Do App{% endblock %}
-
 {% block content %}
   <div class="jumbotron">
     <h1 class="display-4">To-Do App</h1>
     <p class="lead">Just another to-do app.</p>
   </div>
 
   <div class="row">
-    <div class="col-md-12">
+    <div class="col">
       <h2>Items</h2>
       <ul class="list-group mb-4">
-        <!-- List the todo items here -->
+        {% for item in items %}
+          <li>
+            {{ item.title }} -
+            <form action="/toggle?item={{ item.id }}" method="post" class="one-button-form"> 
+              <input type="hidden" name="itemId" value="{{ item.id }}"/>
```

Comment:
It looks like you're passing the item id through twice, both as a query parameter and as a form input; I don't have a strong preference between the two but I think we should remove the duplication since you only use the form value on the backend (for the record, I'd lean towards the form input anyway, simply because that leans into the form structure we have, and saves us constructing parameter strings ourselves - and potentially risking encoding/decoding bugs)